src/bundle/Resources/views/themes/admin/ui/component/custom_dropdown.html.twig

```diff
@@ -0,0 +1,24 @@
+<div class="ez-custom-dropdown__wrapper {% if class is defined %}{{ class }}{%endif%}" {% if source_selector is defined %}data-source-selector="{{ source_selector }}"{%endif%}>
```

Comment:
couldn't it be just `class|default('')` insted of `{% if class is defined %}{{ class }}{%endif%}` ?